src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -1520,24 +1520,34 @@
     short-summary: List custom VM images.
 """
 
+helps['image update'] = """
+    type: command
+    short-summary: Update custom VM images.
+    examples:
+        - name: Add or update a tag.
+          text: az image update -n ImageName -g ResourceGroup --set tags.tagName=tagValue
```

Comment:
Seems very odd to have an example for how to set tags with generic update when you could easily add the `--tags` convenience argument, which would be more consistent with nearly all other updates in the CLI.